packages/libllvm/build.sh

```diff
@@ -96,16 +89,17 @@ termux_step_pre_configure() {
 		termux_error_exit "Invalid arch: $TERMUX_ARCH"
 	fi
 	# see CMakeLists.txt and tools/clang/CMakeLists.txt
-	TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=" -DLLVM_DEFAULT_TARGET_TRIPLE=$LLVM_DEFAULT_TARGET_TRIPLE"
-	TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=" -DLLVM_TARGET_ARCH=$LLVM_TARGET_ARCH -DLLVM_TARGETS_TO_BUILD=all"
+	TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=" -DLLVM_TARGET_ARCH=$LLVM_TARGET_ARCH"
 	TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=" -DLLVM_HOST_TRIPLE=$LLVM_DEFAULT_TARGET_TRIPLE"
 }
+
 termux_step_post_make_install() {
 	if [ $TERMUX_ARCH = "arm" ]; then
-		cp ../src/projects/openmp/runtime/exports/common.min/include/omp.h $TERMUX_PREFIX/include
+		cp $TERMUX_PKG_SRCDIR/projects/openmp/runtime/exports/common.min/include/omp.h $TERMUX_PREFIX/include
 	else
-		cp ../src/projects/openmp/runtime/exports/common.min.ompt.optional/include/omp.h $TERMUX_PREFIX/include
-	fi	
+		cp $TERMUX_PKG_SRCDIR/projects/openmp/runtime/exports/common.min.ompt.optional/include/omp.h $TERMUX_PREFIX/include
```

Comment:
I am getting an error here when I try to build locally for aarch64. omp.h is at 
`/home/builder/.termux-build/libllvm/src/projects/openmp/runtime/exports/common.ompt.optional/include/omp.h` 
instead of 
`/home/builder/.termux-build/libllvm/src/projects/openmp/runtime/exports/common.min.ompt.optional/include/omp.h`
Seems to work for the CI though, weird

Reply:
I changed nothing for this other than adding the prefix `$TERMUX_PKG_SRCDIR` to use the absolute path. I only built this package for AArch64 on Arch linux x86_64, didn't build for ARM myself to make sure that separate path was right.

Reply:
Yeah i have compiled with both. On old docker image it worked, on new it doesn't. So its nobodies fault.